scipy/stats/_multivariate.py

```diff
@@ -34,14 +34,14 @@
 
 
 _doc_random_state = """\
-random_state : {None, int, np.random.RandomState, np.random.Generator}, optional
+seed : {None, int, np.random.RandomState, np.random.Generator}, optional
     Used for drawing random variates.
-    If `seed` is `None` the `~np.random.RandomState` singleton is used.
+    If `seed` is `None`, the `~np.random.RandomState` singleton is used.
     If `seed` is an int, a new ``RandomState`` instance is used, seeded
```

Comment:
Any guesses why we're using double-backticks around RandomState on this line and single on the line above?

Reply:
It's driving my crazy, but perhaps we should leave that to a different issue.

Reply:
> Any guesses why we're using double-backticks around RandomState on this line and single on the line above?

I don't know.

> It's driving my crazy, but perhaps we should leave that to a different issue

Haha, yes. Let's do that.